src/ctia/auth.clj

```diff
@@ -16,62 +16,71 @@
   #{;; Actor
     :create-actor
     :read-actor
+    :list-actors-by-external-id
```

Comment:
I think you should just add one capability called :external-id, and then compose it with the proper :read-<entity> capabilities (eg ":capabilities #{:read-actor :external-id}").  Notice that there are no other :list-<thingy> style capabilities here now.  I'm not necessarily against :list-<thingy> style capabilities, but I think we should prefer composing simpler capabilities when practical.